std/complex.d

```diff
@@ -557,13 +557,13 @@ if (isFloatingPoint!T)
     rcheck = (-a) ^^ 2.0;
     assert(feqrel(rer2.re, rcheck) == double.mant_dig);
     assert(isIdentical(rer2.re, rcheck));
-    assert(approxEqual(rer2.im, 0.0, EPS));
+    assert(isClose(rer2.im, 0.0, 0.0, 1e-10));
 
     auto rer3 = (-a) ^^ complex(-2.0, 0.0);
     rcheck = (-a) ^^ (-2.0);
     assert(feqrel(rer3.re, rcheck) == double.mant_dig);
     assert(isIdentical(rer3.re, rcheck));
-    assert(approxEqual(rer3.im, 0.0, EPS));
+    assert(isClose(rer3.im, 0.0, 0.0, EPS));
```

Comment:
Whenever a comparison to zero is applied, the releative difference check does not work anymore. I disabled it with a default of `0.0` and instead used `EPS` for the absolute difference check, which is appropriate here. I did this at several other places too.